src/Message/RefundRequest.php

```diff
@@ -30,15 +27,12 @@ public function getData()
     }
 
     /**
-     * {@inheritdoc}
-     *
-     * @throws RequestException
-     * @throws InvalidArgumentException
```

Comment:
The interface still does not declare these exceptions, why have they been removed here?